assets/js/modules/analytics/setup/setup-form.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Analytics Setup form.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Button from '../../../components/button';
+import { STORE_NAME } from '../datastore/constants';
+import {
+	AccountSelect,
+	ExistingTagNotice,
+	ProfileSelect,
+	PropertySelect,
+} from '../common/';
+const { useSelect, useDispatch } = Data;
+
+export default function SetupForm( { finishSetup } ) {
```

Comment:
This is amazing. It's so easy-to-read now!

![high-five](https://user-images.githubusercontent.com/90871/79504449-d17f8b00-802a-11ea-90fb-009b5ac07b38.gif)